docs/vc-http-api.yml

```diff
@@ -113,6 +113,65 @@ paths:
           description: invalid input!
         "500":
           description: error!
+  /presentations/available:
+    post:
+      tags:
+        - Holder
+      summary: Notifies a holder of an available presentation.
+      operationId: notifyPresentationAvailable
+      description:
+        This API is the first part of a holder to holder credential exchange over http.
+        The server MAY reply with a query for the client to proceed with a presentation.
+      requestBody:
+        description: Details the client provides to the server to help it decide if the presentation should be made.
+        content:
+          application/json:
+            schema:
+              $ref: "#/components/schemas/NotifyPresentationAvailableRequest"
+
+      responses:
+        "200":
+          description: Proceed with presentation
+          content:
+            application/json:
+              schema:
+                $ref: "#/components/schemas/NotifyPresentationAvailableResponse"
+        "400":
+          description: Request for presentation is malformed
+        "501":
+          description: Not implemented
+        "500":
+          description: internal error
+  /presentations/submissions:
+    post:
+      tags:
+        - Holder
+      summary: Provide a presentation for a holder to store.
+      operationId: storePresentation
+      description: Present from a client holder to a server holder, for the purpose of verification AND storage.
```

Comment:
There are confusing terms here; are they callers/holders/clients/this/server? I am fine will "client holder" and "server holder" - although not sure why the server needs to be a Holder - is it a requirement of this API endpoint that the server _not_ verify the presentation?

Attempt at clearer spec:
"Delivery endpoint for a caller to provide a presentation in compliance with the Verifiable Presentation Request provided in the Notification call. This holder (server) MUST match the 'domain' and 'challenge' to a 'domain' and 'challenge' previously send in a Notification Response, and not already received."

Any other processing rules here? what about the caller-holder's DID?

Reply:
@mavarley the entire point of this API is that the server will be a holder.

The client is sending data to the server that the server is expected to store.
There are no other APIs that explicitly indicate that the server will store / be a holder.

> although not sure why the server needs to be a Holder - is it a requirement of this API endpoint that the server not verify the presentation?

I would see verification as a strong SHOULD for this API, but its clearly not a must like the verifyPresentation.

Again, please convert this suggestion to something I can approve or an issue.

Reply:
```suggestion
      description: "Delivery endpoint for a client-holder to provide a presentation in compliance with the Verifiable Presentation Request provided in the Notification call. This server-holder MUST match the 'domain' and 'challenge' to a 'domain' and 'challenge' previously sent in a Notification Response, and not already received."
```